employeelist/src/main/java/pro/sky/exever/employeelist/DepartmentController.java

```diff
@@ -0,0 +1,53 @@
+package pro.sky.exever.employeelist;
+
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import org.springframework.http.HttpStatus;
+
+@RestController
+@RequestMapping("/departments")
+public class DepartmentController {
+	private final DepartmentService departmentService;
+	private static final Logger log = Logger.getLogger("pro.sky.exever.employeelist.EmployeeController");
```

Comment:
если поле static и final, то называть его нужно в UPPER_CASE
```suggestion
	private static final Logger LOG = Logger.getLogger("pro.sky.exever.employeelist.EmployeeController");
```

Reply:
Также чтобы каждый раз не исправлять полный путь к классу при перемещении его по пакетам лучше просто указывать класс в методе getLogger
```suggestion
	private static final Logger log = Logger.getLogger(DepartmentController.class);
```